pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -114,8 +114,19 @@ static void WipeSettings()
 	wxGetApp().CleanupRestartable();
 	wxGetApp().CleanupResources();
 
-	wxRemoveFile(GetUiSettingsFilename());
-	wxRemoveFile(GetVmSettingsFilename());
+	try
```

Comment:
Non-throwing overloads can be used, removing the need for this try...catch block.